3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -231,7 +236,14 @@ object EnrichmentManager {
       event.page_urlquery = components.query.orNull
       event.page_urlfragment = components.fragment.orNull
     }
-
+    
+    // For Currency Conversion Enrichment
+    val trTotal = (CU.stringToDoublelike("tr_total", "tr_tt" )).toString
```

Comment:
The good news is that the errors from CU.stringToDoublelike are already reported in the overall success or failure of the EnrichmentManager, so you just have to worry about the happy path (you can do a pattern match to pull out the good values).